doc/ChangeLog

```diff
@@ -1,4 +1,8 @@
 
+2022-07-28  Nicolas Berthier <nicolas.berthier@ocamlpro.com>
```

Comment:
I suggest to combine this and the next Changelog entry as `-fformat` is new in any case, maybe go with the current date?

[that's no request, more the way I'd do it]